Makefile

```diff
@@ -268,6 +273,14 @@ build-ubuntu-coverage:
 	docker build -t antrea/antrea-ubuntu-coverage:$(DOCKER_IMG_VERSION) -f build/images/Dockerfile.build.coverage .
 	docker tag antrea/antrea-ubuntu-coverage:$(DOCKER_IMG_VERSION) antrea/antrea-ubuntu-coverage
 
+.PHONY: build-scale-simulator
+build-scale-simulator:
+	@echo "===> Building simulator bin and antrea-ubuntu-simulator image"
+	docker build -t harbor-repo.vmware.com/dockerhub-proxy-cache/antrea/antrea-ubuntu-simulator:$(DOCKER_IMG_VERSION) \
+	    -f build/images/Dockerfile.simulator.build.ubuntu .
+	docker tag harbor-repo.vmware.com/dockerhub-proxy-cache/antrea/antrea-ubuntu-simulator:$(DOCKER_IMG_VERSION) \
+	    harbor-repo.vmware.com/dockerhub-proxy-cache/antrea/antrea-ubuntu-simulator
```

Comment:
`harbor-repo.vmware.com/dockerhub-proxy-cache/antrea/antrea-ubuntu-simulator` doesn't seem right for 2 reasons:
* `harbor-repo.vmware.com` is a private VMware registry, not accessible outside of the VMware corporate network
* `dockerhub-proxy-cache` is meant to be used as a cache to pull images; it doesn't make sense to build an image locally and tag it like this

Reply:
thanks, should I push dockerhub first ,or use projects.registry.vmware.com/ in https://github.com/vmware-tanzu/antrea/pull/1617?

Reply:
If you edit https://github.com/vmware-tanzu/antrea/blob/master/.github/workflows/build.yml, then an updated image for the simulator will be pushed to dockerhub every time master is updated. This image will be accessible through the VMware internal dockerhub proxy cache (if you are on the corporate network) AND will automatically be mirrored to `projects.registry.vmware.com` where everyone will be able to pull it with no rate limiting restrictions. The YAML files can reference `projects.registry.vmware.com` as it is a public registry.

Reply:
@antoninbas thanks, just change it to antrea/antrea-simulator-ubuntu